### DISCUSSION TITLE
Multiple checkboxes bind to array, fast clicks

### DISCUSSION THREAD

**Discussion by ThibaudDauce** [livewire/livewire#3220] [7. Bugs (archived)]
Hi,

If I have two checkboxes bind to the same array and I quickly click the two checkbox the second update request will override the first one and only the second checkbox will be checked. Is there a way (a modifier?) to tell Livewire to wait the end of the state update before updating the array?

It seems that the second request is buffered with an update containing only the second value, I would like to wait the first state update so that the second request will get send the first and the second value to the PHP.

**Comment by rahmanramsi** [comment]
did u already try to add defer to model? so it not instantly request after get clicked

**Comment by joshhanley** [comment]
@ThibaudDauce there is also a `.debounce`, not sure if it works on checkboxes or just text inputs, but worth experimenting.

Hope this helps!

**Comment by younus93** [comment]
@ThibaudDauce Were you able to fix this issue? I tried both debounce / lazy and the issue still persists.
